aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -203,7 +204,11 @@ void gemm<at::Half>(CUDABLAS_GEMM_ARGTYPES(at::Half)) {
 #else
   cudaDeviceProp* prop = at::cuda::getCurrentDeviceProperties();
   if (prop->major >= 5) {
+#if defined(CUDA_VERSION) && CUDA_VERSION < 11000
+    // On CUDA 11, tensor core will be automatically used with default math type,
+    // so there is no need to change the math type.
```

Comment:
Math mode might not be `CUBLAS_DEFAULT_MATH` here, it might be `CUBLAS_TF32_TENSOR_OP_MATH`.  But both allow tensor cores for fp16 inputs.  Can we change the comment to
```
// On Cuda 11, tensor cores are allowed by default for FP16 inputs if the math mode is
// `CUBLAS_DEFAULT_MATH` or `CUBLAS_TF32_TENSOR_OP_MATH`, so there's no need to change
// the math mode locally.
```

Reply:
Sure, good idea